### DISCUSSION TITLE
Unable to setup development server

### DISCUSSION THREAD

**Discussion by Sulochan-khadka** [coder/code-server#6923] [General]
I wanted to "try" some features, but am unable to start development server. All i get is this error: 

ENOENT: no such file or directory, open 'code-server/lib/vscode/out/vs/server/node/server.main.js'

The node folder is empty in my workspace. Is that why it is throwing error? -- Help me out.

**Comment by code-asher** [comment]
Usually this means VS Code has not finished compiling, so you probably just need to keep waiting.  It can take quite a long time (several minutes).
